infoschema/infoschema.go

```diff
@@ -279,13 +281,14 @@ const (
 	codeForeignKeyNotExists = 1091
 	codeWrongFkDef          = 1239
 
-	codeDatabaseExists  = 1007
-	codeTableExists     = 1050
-	codeBadTable        = 1051
-	codeColumnExists    = 1060
-	codeIndexExists     = 1831
-	codeMultiplePriKey  = 1068
-	codeTooManyKeyParts = 1070
+	codeDatabaseExists      = 1007
+	codeTableExists         = 1050
+	codeBadTable            = 1051
+	codeColumnExists        = 1060
+	codeIndexExists         = 1831
+	codeMultiplePriKey      = 1068
+	codeTooManyKeyParts     = 1070
+	codeUnknownCharacterSet = 1115
```

Comment:
Replace this by `codeUnknownCharacterSet = mysql.ErrUnknownCharacterSet`, By the way, you can modidy all above errCode to use mysqlErrxxx